packages/ember-routing-htmlbars/lib/keywords/closure-action.js

```diff
@@ -67,6 +67,22 @@ export default function closureAction(morph, env, scope, params, hash, template,
   return s;
 }
 
+
+function extractRawAction(params, scope) {
+  var [value] = params;
+  if (value) {
```

Comment:
You are indeed correct, I think https://github.com/emberjs/ember.js/blob/master/packages/ember-htmlbars/lib/hooks/get-root.js#L22-L25 would cause that...